<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="X8002009" name="Academy.Validation" tests="3" failures="0" timestamp="2020-05-23T16:18:10 CDT" time="0.019" errors="1">
  <testcase name="headerValidation" time="0.000" classname="Academy.Validation">
    <skipped/>
  </testcase> <!-- headerValidation -->
  <system-out/>
  <testcase name="validateCourses" time="0.000" classname="Academy.Validation">
    <skipped/>
  </testcase> <!-- validateCourses -->
  <system-out/>
  <testcase name="intialization" time="0.019" classname="Academy.Validation">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at resources.base.invokeBrowser(base.java:29)
at Academy.Validation.intialization(Validation.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:619)
at org.testng.TestRunner.beforeRun(TestRunner.java:609)
at org.testng.TestRunner.run(TestRunner.java:580)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- intialization -->
  <system-out/>
</testsuite> <!-- Academy.Validation -->
